### DISCUSSION TITLE
hc:Poptip气泡怎么设置提示文字的颜色

### DISCUSSION THREAD

**Discussion by 201705110418** [HandyOrg/HandyControl#786] [Q&A] [answered]

**Comment by 201705110418** [comment, marked as answer]
`<Button Style="{StaticResource ButtonPrimary}"  Content="example" >
                            <hc:Poptip.Instance>
                                <hc:Poptip PlacementType="TopLeft" Foreground="Gray" hc:Poptip.Content="test"/>
                            </hc:Poptip.Instance>
                        </Button>`

**Reply by 201705110418** [reply]
我已经解决了这个问题，请问如何关闭讨论？

**Reply by NaBian** [reply]
如果解决了，可以在这里留下解决方案
